db.js

```diff
@@ -449,21 +449,56 @@ var loans = (function () {
       let updateInserts = [loan.status, loan.id];
       updateQuery = mysql.format(updateQuery, updateInserts);
 
-      //var now = moment().format("YYYY-MM-DD HH:mm:ss");
-      //will be used later for timestamps
-      //console.log(now);
-
-      return pool.query(updateQuery).then((updateResult) => {
-        if (loan.status === "accepted") {
-          let deleteQuery =
-            "DELETE FROM loans WHERE status = 'pending' AND book_id = ?";
-          let deleteInserts = [loan.book_id];
-          deleteQuery = mysql.format(deleteQuery, deleteInserts);
-
-          pool.query(deleteQuery);
-        }
-        return updateResult;
-      });
+      var currentDate = moment().format("YYYY-MM-DD HH:mm:ss");
```

Comment:
Is there a reason to use moment rather than a simple timestamp like `Date.now()`? From what I understand, Date is built into JavaScript and moment is a heavier weight library for parsing dates and displaying them in the correct locale (i.e. English vs. Spanish month/day order). A timestamp is less human readable if you're viewing the raw database values but may be easier to parse on the frontend.

Reply:
I didn't use `Date.now()` because that doesn't directly output the timestamp in the way the database accepts and I didn't see the point in doing a ton of string manipulation if a library could do so. 

And I'm down to change the timestamp to something less specific or more readable, but that would require changing the database schema. I would actually prefer a less complex time value for the timestamp fields in the loan table, so I'd be down to loosen it up a little if need be.

Reply:
You could use dayjs, which is a much smaller library that accomplishes the same thing

Reply:
I tried installing daysjs with both node and yarn, which resulted in two errors each time. On the node installation, it said my OS rejected it, and the yarn installation said something similar. I also attempted to run it as sudo, which failed similarly. 

I think it might be best to stick with what we know will work, regardless if moment.js is a slightly heavier library. I mainly say that because moment.js isn't always running or consuming processing power, so I don't think it would be a hindrance to keep it in.

Reply:
Ok, I didn't realize that the database needed that specific format.